components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/handler/UserEmailVerificationHandler.java

```diff
@@ -332,11 +340,31 @@ private void triggerNotification(User user, String type, String code, Property[]
         }
         properties.put(IdentityRecoveryConstants.TEMPLATE_TYPE, type);
         Event identityMgtEvent = new Event(eventName, properties);
+
+        String action;
+        if (recoveryDataDO !=null && recoveryDataDO.getRecoveryScenario()
+                == RecoveryScenarios.ADMIN_FORCED_PASSWORD_RESET_VIA_EMAIL_LINK) {
+            action = AuditConstants.ACTION_NOTIFICATION_FOR_FORCED_PASSWORD_REST_VIA_EMAIL_LINK;
+        } else if (recoveryDataDO !=null && recoveryDataDO.getRecoveryScenario()
```

Comment:
```suggestion
        } else if (recoveryDataDO ! =null && recoveryDataDO.getRecoveryScenario()
```

Reply:
Fixed in https://github.com/wso2-extensions/identity-governance/pull/452/commits/a9e205abd2e323e9cdf572f939e8fb0321757f12